src/main/java/Deadline.java

```diff
@@ -0,0 +1,10 @@
+public class Deadline extends Task{
+    protected String by;
+    public Deadline(String description, String by){
+        super(description);
+        this.by = by;
+    }
+    public String toString(){
```

Comment:
I think we need to have a blank line in between variables declared and method, as well as between two methods.